test/snippet/io/sequence_file/sequence_file_output_col_based_writing.cpp

```diff
@@ -0,0 +1,26 @@
+#include <sstream>
+#include <string>
+#include <tuple>
+
+#include <seqan3/alphabet/nucleotide/dna4.hpp>
+#include <seqan3/io/sequence_file/output.hpp>
+#include <seqan3/range/container/concatenated_sequences.hpp>
+
+using seqan3::operator""_dna4;
+
+struct data_storage_t
+{
+    seqan3::concatenated_sequences<seqan3::dna4_vector> sequences{"ACGT"_dna4, "AAA"_dna4};
+    seqan3::concatenated_sequences<std::string> ids{std::string{"ID1"}, std::string{"ID2"}};
+};
+
+data_storage_t data_storage{}; // a global or globally used variable in your program
```

Comment:
No, we should not teach to use non-const global data in snippets. That is not recommended. Can you please make it a local variable of the main method. Still this can hold data used globally throughout the program.

Reply:
And of course other places if this still occurs. Sorry, I didn't see that the first time.